src/components/RoverList.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, Fragment } from 'react';
+import { Header, Loader } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+class RoverList extends Component {
+  componentDidMount() {
+    const { fetchRovers } = this.props;
+    fetchRovers();
+  }
+
+  render() {
+    const { rovers } = this.props;
+    return (
+      <Fragment>
+        {
+          rovers === null
+            ? (<Loader active />)
+            : (
+              <Header>
+                {`Welcome to rovercode, you have ${rovers.length} rover(s)!`}
```

Comment:
After trying this, it looks like this needs to be `rovers.data.length`:
![image](https://user-images.githubusercontent.com/1184314/43808540-85e5a864-9a7b-11e8-995f-224310d0376e.png)

Reply:
I pretty much just mocked what I thought the API would return, still working on getting docker to play nicely... I used a version installed via brew & that seemed to work just fine, but somehow I think the ports aren't being exposed properly, because `lsof -i :8000` shows jack diddly while the django app is running

Reply:
^ yup, my local docker was the issue.  now the containers are up and running and the API responding 👍